core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -413,7 +413,7 @@ export class CreateServiceComponent implements OnInit {
     this.rateExpression.type = 'none';
     this.rateExpression.duration = "5";
     this.eventExpression.type = 'awsEventsNone';
-    this.runtime = 'nodejs';
+    this.runtime = this.runtimeKeys[0];
```

Comment:
Good catch! I see one more line in this file  where nodejs is hardcoded. Can we remove it too? 
https://github.com/tmobile/jazz/blob/d7b8127c8725b699ad011cc4c151f26376d09553/core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts#L39

Reply:
The above line is before the initialization of `runtimeObject `. So I get the value from `envLists` object from environment file directly.